cli/Cargo.toml

```diff
@@ -24,8 +24,8 @@ harness = false
 path = "./bench/main.rs"
 
 [build-dependencies]
-deno_core = { path = "../core", version = "0.81.0" }
-deno_runtime = { path = "../runtime", version = "0.9.3" }
+deno_core = { path = "../core", version = "0.82.0" }
+deno_runtime = { path = "../runtime", version = "0.9.4" }
```

Comment:
This should be a minor bump. If a dependency a crate depends on is bumped with a minor, and the dependent crate reexports that dependency, then the dependent crate _must_ also be bumped a minor release, otherwise we break semver.

Reply:
Bumped `deno_runtime` to `0.10.0`